C-Plus-Plus/other/BrianKenighan'sAlgorithm.cpp

```diff
@@ -0,0 +1,60 @@
+// This is a preferred approach to count no. of set bits of a digit.
+
+//  Brian Kernighan’s Algorithm:
+<<<<<<< HEAD
+=======
+// Concept -> If we subtract 1 from a decimal number, it will flip all the bits
+// after the rightmost set bit i.e 1 including that rightmost bit too.
+// As 10 is 0000 1010, 9 is 0000 1001
+// you can see in above example rightmost set bit in 10 is at second position
+// therefore in number (10 - 1) = 9 that second position and after that 1st position
+// flips away. 
+// So, we do bitwise of n and (n-1), then we unset the rightmost unset bit and we set
+// a counter to count the number of times loop executes we set bit count.   
+// In brief, no. of times loop runs is equal to count of set bits.
+>>>>>>> 562da050f4cd780afd27ff67ca30ce7ef408cb26
+
+/* Concept -> If we subtract 1 from a decimal number, it will flip the all bits after the rightmost set bit i.e 1 including that rightmost bit too.
+ As 10 is 0000 1010, 9 is 0000 1001, you can see in this example rightmost set bit in 10 is at second position
+ therefore in number (10 - 1) = 9 that second position and after that 1st position
+ flips away. So, we do bitwise of n and (n-1), then we unset the rightmost unset bit and we set
+ a counter to count the number of times loop executes we set bit count.   
+ In brief, no. of times loop runs is equal to count of set bits.
+*/
```

Comment:
Justify this content

Reply:
This content explains the approach to solve the PS.

Reply:
The content should be justify aligned. Justify alignment means "when you justify text, space is added between words so that both edges of each line are aligned with both margins."